services/outputhost/messagecache.go

```diff
@@ -1206,12 +1212,10 @@ func (msgCache *cgMsgCache) logStateMachineHealth() {
 	}).Info(`state machine health`)
 }
 
-var logPumpHealthOnce bool
+var logPumpHealthOnce sync.Once
```

Comment:
This one is meant to be global; when we have different capacities per cgMsgCache, then this will make sense.